packages/replace/rollup.config.js

```diff
@@ -0,0 +1,12 @@
+import buble from 'rollup-plugin-buble';
```

Comment:
Just a thought here, doesn't affect this PR, but with it being a mono repo, one of the huge positives is that plugins are all tested with each other for compatibility and breaking changes. Do we want to consider actually importing the plugin from the monorepo once they are migrated over?